docs/sources/configure/options.md

```diff
@@ -502,6 +502,31 @@ attributes:
       exclude: ["k8s.pod.*"]
 ```
 
+Additionally, you can use "`*`" wildcards as metric names to add and exclude attributes for
+groups of metrics having the same name. For example:
+
+```yaml
+attributes:
+  select:
+    http_*:
+      include: ["*"]
+      exclude: ["http_path", "http_route"]
+    http_client_*:
+      # override http_* exclusion
+      include: ["http_path"]
+    http_server_*:
+      # override http_* exclusion
+      include: ["http_route"]
+```
+
+In the previous example, all the metrics with a name starting with `http_` (or `http.`) would include all
```

Comment:
Could we word it as follows:

> In the example, attributes starting with http_` are excluded and all other attributes are included.

Why have you included an alternative format with a full stop e.g. `http.`, is there some conversion logic to make them equivalent?

Reply:
In that case, the `http_*` does not refer to attribute names but to the metric names. So your suggestion wouldn't be accurate.

Since Beyla supports either the Prometheus naming of metrics and attributes (underscore-based) and the OpenTelemetry form (dot-based) I specified both forms, but since these forms are documented already in https://grafana.com/docs/beyla/latest/metrics/ I could maybe just add a paragraph saying something like:

> The metrics and attribute names accept either the Prometheus form (`underscore_based`) and the OpenTelemetry format (`dot.based`).

Reply:
I like you suggestion explaining the supported formats, let's go with that 👍

Reply:
I tried to simplify the logic, does my sentence make, sense?

> in the example, attributes starting with http_` are excluded and all other attributes are included.

Reply:
In the example, the `http_*` does not refer to attributes but to metric names. Maybe we can change the example to make it simpler to explain. something like:

```yaml
attributes:
  select:
    http_*:
      include: ["*"]
      exclude: ["http_path", "http_route"]
    http_client_request_duration:
      # override http_* exclusion
      include: ["http_path"]
```

> In the example, most metrics starting with `http_` would include all the attributes but `http_path` and `http_route`. With the exception of the `http_client_request_duration` metric, which overrides the general configuration to allow the `http_path` attribute for this specific metric.

Reply:
Ok let's go with that.

Reply:
Let's keep the original for now and we can add further examples when we split out the options page as there is a lot of content.